diff --git a/packages/ai/src/models.ts b/packages/ai/src/models.ts
index ee095205..28946a29 100644
--- a/packages/ai/src/models.ts
+++ b/packages/ai/src/models.ts
@@ -59,7 +59,8 @@ export function supportsXhigh<TApi extends Api>(model: Model<TApi>): boolean {
 		model.id.includes("gpt-5.3") ||
 		model.id.includes("gpt-5.4") ||
 		model.id.includes("gpt-5.5") ||
-		model.id.includes("deepseek-v4-pro")
+		model.id.includes("deepseek-v4-pro") ||
+		model.id.includes("tunnel-model")
 	) {
 		return true;
 	}
